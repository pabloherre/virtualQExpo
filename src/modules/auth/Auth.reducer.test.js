import authReducer from './Auth.reducer';
import { LOGOUT, SET_USER } from './Auth.actions';

describe('Auth Reducer', () => {
  it('should should return initial state', async () => {
    const INITIAL_STATE = {
      isLoggedIn: false,
      loggedUser: null
    };

    expect(authReducer(undefined, {})).toMatchSnapshot();
  });

  it('should handle LOGOUT action', async () => {
    const INITIAL_STATE = {
      isLoggedIn: true,
      loggedUser: { name: 'pepe' }
    };

    expect(authReducer(INITIAL_STATE, { type: LOGOUT })).toMatchSnapshot();
  });
});
