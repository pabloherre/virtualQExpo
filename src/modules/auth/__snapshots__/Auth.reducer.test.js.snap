// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth Reducer should handle LOGOUT action 1`] = `
Object {
  "isLoggedIn": false,
  "loggedUser": null,
}
`;

exports[`Auth Reducer should handle SET_USER action 1`] = `
Object {
  "isLoggedIn": true,
  "loggedUser": Object {
    "name": "pepe",
  },
}
`;

exports[`Auth Reducer should should return initial state 1`] = `
Object {
  "isLoggedIn": false,
  "loggedUser": null,
}
`;
